android-components/components/ui/tabcounter/src/main/res/drawable/mozac_ic_mask.xml

```diff
@@ -0,0 +1,26 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="oval">
+            <size
+                android:width="16dp"
```

Comment:
Wondering if this should be 20x20 to align with the [figma](https://www.figma.com/file/Ha2T4I0nX0h7B95GYxH6wV/v1-ABM-minimal-UI?type=design&node-id=2585-14916&mode=design&t=UMVacF6H6bTpiV7e-0)?

<img width="1726" alt="Screen Shot 2023-08-16 at 9 36 30 PM" src="https://github.com/mozilla-mobile/firefox-android/assets/1190888/be494b09-df8b-4eb9-a01b-14e8e79c06b7">